### ISSUE TITLE
Kernel died when importing the 'pycuda.autoinit' module on win10?

### ISSUE THREAD

**Issue by cuixing158** [inducer/pycuda#249]
my os: win10
python: 3.7

Kernel died when importing the pycuda.autoinit module on win10? I installed pycuda through pip, how can I solve it?
![pycuda](https://user-images.githubusercontent.com/22857777/96210555-bf17e700-0fa4-11eb-8758-6e572c8fbf10.jpg)
![pycuda2](https://user-images.githubusercontent.com/22857777/96210556-bfb07d80-0fa4-11eb-8c59-4499aaeb9306.jpg)

**Comment by inducer** [comment]
Please supply console output and a stack trace of the crashed process. Does pycuda work if used from the command line? Can you run CUDA C examples from the CUDA SDK?

**Comment by cuixing158** [comment]
No,it does't work, When I entered the console, there was no output after waiting for a few seconds, and I quit to the command line directly.
![pycuda](https://user-images.githubusercontent.com/22857777/96263905-2ad67000-0ff6-11eb-849c-f9c0621de1f9.jpg)

 In addition, I wrote a simple sample program, and then executed it from the console, `python testpycuda.py` waits a few seconds without any output printing, automatically exit the python environment?
```python
import pycuda.driver as drv
import pycuda.tools
import pycuda.autoinit
import numpy
import numpy.linalg as la
from pycuda.compiler import SourceModule

mod = SourceModule("""
__global__ void multiply_them(float *dest, float *a, float *b)
{
  const int i = threadIdx.x;
  dest[i] = a[i] * b[i];
}
""")

multiply_them = mod.get_function("multiply_them")

a = numpy.random.randn(400).astype(numpy.float32)
b = numpy.random.randn(400).astype(numpy.float32)

dest = numpy.zeros_like(a)
multiply_them(
        drv.Out(dest), drv.In(a), drv.In(b),
        block=(400,1,1))

print(dest-a*b)
```

**Comment by inducer** [comment]
It's hard to diagnose that given that there appears to be no error message whatsoever. Does the CUDA driver API example from the CUDA SDK work when run? Another thing you could try is to rebuild PyCUDA with tracing (`./configure.py --cuda-trace`, `deltree build`, `python setup.py install`) to see which call is ending your program.

**Comment by tireira** [comment]
Does anyone find a solution to this issue? 
I have the same problem. I have a python 3.6.13 version and working on windows 10. 
The kernel crashes and i have this kind of exit status : 
![image](https://user-images.githubusercontent.com/3064194/149328613-e56711ff-4457-4bb2-a5d3-68761b04eaa6.png)
Thank you in advance for any help.

**Comment by tireira** [comment]
The  program crashes at  return ctx_marker(dev) 
![image](https://user-images.githubusercontent.com/3064194/149351232-5c3eb90f-74e7-4313-baf2-f12cb4908601.png)
in the file tools.py. 
is it due to virtual studio version ?
